components/PaginationLinks.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
```

Comment:
We actually don't need to import React when using JSX in a Next.js project. (We need it, though, if `React` variable is used.) So let's remove this.